Tests/Shopware/Connect/Component/CategoryExtractorTest.php

```diff
@@ -650,5 +650,51 @@ public function testGetRemoteCategoriesTreeByStream()
         $remoteCategories = $this->categoryExtractor->getRemoteCategoriesTreeByStream($stream, $shopId);
         $this->assertEquals($expected, $remoteCategories);
     }
+
+    /**
+     * Test concat shopId, categoryId and unique number
+     */
+    public function testConcatShopIdAndCategoryId()
+    {
+        $shopId = 1;
+
+        $randomStringGenerator = $this->getMockBuilder('\\ShopwarePlugins\\Connect\\Components\\RandomStringGenerator')
```

Comment:
Shopware 5.2 has a requirement > php5.6, so you could use `RandomStringGenerator::class`. And you can use the short array syntax :)